### ISSUE TITLE
[FEATURE REQUEST] Virtual scroll for swipeList

### ISSUE THREAD

**Issue by CesarGomezTissini** [eCollect/vue-swipe-actions#11]
It could be nice if we have a option to virtual scrolling a swipeList, at least limit the number of visible items per list.

**Comment by nanov** [comment]
Hi, an interesting feature, but I'm afraid it is out of scope of this component.

The SwipeList component is just a sugar/convince wrapper around the SwipeOut component which provides the swipe functionality. A virtual list may be achieved combining the SwipeOut component together with one of the available virtual list components ( [this](https://github.com/Akryum/vue-virtual-scroller) or [this](https://github.com/tangbc/vue-virtual-scroll-list) for example ).

For inspiration how this may be achieved you can take a look at the the [SwipeList](https://github.com/eCollect/vue-swipe-actions/blob/master/src/components/SwipeList.vue) component. Should be pretty straight forward.

**Comment by CesarGomezTissini** [comment]
Yeah, I made a custom implementation, but I think it could be nice for future feat.

Thanks anyway 🗡
